core/.planning/phases/02-attribute-filter-implementation/02-02-PLAN.md

```
@@ -0,0 +1,227 @@
+  key_links:
+    - from: "AttributeSqlGenerator"
+      to: "SqlFragment"
+      via: "Every operator returns SqlFragment"
+      pattern: "SqlFragment\\("
+    - from: "AttributeSqlGenerator"
+      to: "ParameterNameGenerator"
+      via: "Uses generator for unique parameter names"
+      pattern: "paramGen\\.next"
+---
+
+<objective>
```

markdownlint-cli2 (0.20.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)